vector-app/src/gplay/java/im/vector/app/di/FlavorModule.kt

```diff
@@ -46,6 +50,12 @@ abstract class FlavorModule {
     @Binds
     abstract fun bindsFcmHelper(fcmHelper: GoogleFcmHelper): FcmHelper
 
+    @Binds
```

Comment:
I think we need to add those bindings in FDroid variant as well, no (in the dedicated `FlavorModule`)?